ptf_tests/tests/l3_action_profile_with_vhost_ports.py

```diff
@@ -136,41 +136,48 @@ def runTest(self):
                 print(f"FAIL: Verification of packets sent failed with exception {err}")
                 self.PASSED = False
 
-        # verify whether traffic hits group-2
-        for src in self.config_data['traffic']['in_pkt_header']['ip_src']:
+        # sender_vm_id, receiver_vm_id = 0,1
+        # Get send_port_id, receive_vm_id
+
+        sender_vm_id = self.config_data['traffic']['send_port'][1]
+        receiver_vm_id= self.config_data['traffic']['receive_port'][1]
+        sender_conn, receiver_conn = eval("conn"+str(sender_vm_id+1)), eval("conn"+str(receiver_vm_id+1))
+        #verify whether traffic hits group-2/Multicast
+        send_result = test_utils.send_scapy_traffic_from_vm(sender_vm_id, sender_conn, receiver_conn, self.config_data,'multicast')
+        if send_result:
+            result = test_utils.verify_scapy_traffic_from_vm(receiver_vm_id,conn2,self.config_data)
+            if not result:
+                self.result.addFailure(self, sys.exc_info())
+            else:
+                print(f"PASS: Verification of packets for group-2 PASSED on receiver vm_id {receiver_vm_id}")
+        else:
+            print(f"FAIL: Packets not sent from sender_vm with id {sender_vm_id}")
+            self.result.addFailure(self, sys.exc_info())
 
```

Comment:
Instead of group-1 2 and 3, change to unicast, multicast, broadcast so people don't confuse it with action selector group ID.

Reply:
done, added changes to logs